src/util.cpp

```diff
@@ -315,10 +315,11 @@ QString Util::formatTimeString(quint64 nanoseconds, bool shortForm)
 
     const auto milliseconds = (nanoseconds / 1000000) % 1000;
```

Comment:
I think this is then equally wrong in the usage below in loc 340?

please move this variable into the conditional, and do the same for the `microseconds` above in loc 307 to ensure they are not misused. then you can use the same long name `microseconds` here.